### PR TITLE
fix integer vId

### PR DESCRIPTION
**Pull request by critical27**
1. support int vId
2. fix storage can't be stopped
3. add uuid

### CODE REVIEW

File: src/storage/exec/IndexOutputNode.h [hunk 1]
```diff
@@ -276,7 +276,7 @@ class IndexOutputNode final : public RelNode<T> {
 
     void addVidToRow(VertexIDSlice vId, Row& row) {
         if (planContext_->isIntId_) {
-            row.emplace_back(vId.toString());
+            row.emplace_back(*reinterpret_cast<const int64_t*>(vId.data()));
```
Comment (darionyaphet) [inline src/storage/exec/IndexOutputNode.h:279]: So vid will be explained as a string previously?

**Review by darionyaphet — DISMISSED** [review]

File: src/storage/exec/IndexOutputNode.h (hunk 1 above)
Comment (critical27) [reply src/storage/exec/IndexOutputNode.h:279]: Still need to copy, this is to prevent the id is more than expected 8 bytes.

**Review by panda-sheep — APPROVED** [review]
Good，LGTM

**Review by panda-sheep — APPROVED** [review]
LGTM

**Review by bright-starry-sky — APPROVED** [review]
Well done

**Review by nevermore3 — DISMISSED** [review]

**Review by bright-starry-sky — APPROVED** [review]

**Review by panda-sheep — APPROVED** [review]

**Pull request merged**